noxfile.py

```diff
@@ -180,7 +180,7 @@ def run_doctests(session):
         "-m",
         "pytest",
         "--doctest-plus",
-        "pybamm",
+        "src/pybamm",
```

Comment:
```suggestion
        "src",
```

This can be just `src` and you would not need to deal with different slashes for paths (such as on Windows)

Reply:
I think this is still a valid suggestion

Reply:
Sorry must have missed it.